utils/src/ooinstall/cli_installer.py

```diff
@@ -72,7 +72,7 @@ def delete_hosts(hosts):
                 click.echo("\"{}\" doesn't coorespond to any valid input.".format(del_idx))
     return hosts, None
 
-def collect_hosts(oo_cfg, masters_set=False, print_summary=True):
+def collect_hosts(oo_cfg, existing_env=False, masters_set=False, print_summary=True):
```

Comment:
I almost think we need to maintain some type of state.

Basically, to start, we can have the following:
new_host
installed_host
failed_host

And update the state (and persist to the config file) as state changes for a host.
Then, we could have a proper way to re-run against a set of hosts (any that were in a failed_host state)

It'll also lay the ground work for supporting adding additional host types.